modules/appeals_api/config/routes.rb

```diff
@@ -1,11 +1,14 @@
 # frozen_string_literal: true
 
-AppealsApi::Engine.routes.draw do
+AppealsApi::Engine.routes.draw do # rubocop:disable Metrics/BlockLength
```

Comment:
This shouldn't be necessary because the modules routes files are supposed to ignore BlockLength
I'm looking into why the settings in `.rubocop.yml` aren't applying here

Reply:
Ah ha!  AppealsApi has it's own rubocop.yml! `modules/appeals_api/.rubocop.yml`
update that to look like this:
```
Metrics/BlockLength:
  Exclude:
    - spec/**/*.rb
    - 'config/**/*.rb'
```
and then you don't need to disable rubocop here

Reply:
I was also wondering why this was needed here and not other routes files. Thanks and updated!